src/components/HOC/withApi.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+const withApi = (WrappedComponent, charactersQuantity) => {
+    class WithApi extends React.Component {
+        constructor(props) {
+            super(props)
+            this.state = {
+                charactersCount: 0,
+                charactersArr: [],
+                randomIds: [],
+                test: "test works"
+            }
+        }
+        
+        getCharactersArr = (callback) =>{
```

Comment:
Whitespace missing. 
Maybe you could try to incorporate eslint to check such kind of errors for you? :)

Reply:
I have added eslint with air bnb guidelines, also I have added Prettier to project and localy to vs code.